src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -195,27 +195,24 @@ private BoundExpression BindObjectCreationExpression(BoundUnconvertedObjectCreat
             var syntax = node.Syntax;
             switch (type.TypeKind)
             {
+                case TypeKind.Enum:
                 case TypeKind.Struct:
                 case TypeKind.Class when !type.IsAnonymousType: // We don't want to enable object creation with unspeakable types
                     return BindClassCreationExpression(syntax, type.Name, typeNode: syntax, (NamedTypeSymbol)type, arguments, diagnostics, node.InitializerOpt, wasTargetTyped: true);
                 case TypeKind.TypeParameter:
                     return BindTypeParameterCreationExpression(syntax, (TypeParameterSymbol)type, arguments, node.InitializerOpt, typeSyntax: syntax, diagnostics);
                 case TypeKind.Delegate:
                     return BindDelegateCreationExpression(syntax, (NamedTypeSymbol)type, arguments, node.InitializerOpt, diagnostics);
+                case TypeKind.Interface:
+                    return BindInterfaceCreationExpression(syntax, (NamedTypeSymbol)type, diagnostics, typeNode: syntax, arguments, node.InitializerOpt, wasTargetTyped: true);
                 case TypeKind.Array:
-                case TypeKind.Enum:
                 case TypeKind.Class:
+                case TypeKind.Dynamic:
                     Error(diagnostics, ErrorCode.ERR_TypelessNewIllegalTargetType, syntax, type);
                     goto case TypeKind.Error;
-                case TypeKind.Interface:
-                    Error(diagnostics, ErrorCode.ERR_NoNewAbstract, syntax, type);
-                    goto case TypeKind.Error;
                 case TypeKind.Pointer:
                     Error(diagnostics, ErrorCode.ERR_UnsafeTypeInObjectCreation, syntax, type);
                     goto case TypeKind.Error;
-                case TypeKind.Dynamic:
-                    Error(diagnostics, ErrorCode.ERR_NoConstructors, syntax, type);
-                    goto case TypeKind.Error;
                 case TypeKind.Error:
                     return MakeBadExpressionForObjectCreation(syntax, type, arguments, node.InitializerOpt, typeSyntax: syntax, diagnostics);
```

Comment:
❓ Should we fallback to BindClassCreationExpression in error cases? (similar to BindObjectCreationExpression)

Reply:
Looking at `BindClassCreationExpression`, I don't see a benefit. `MakeBadExpressionForObjectCreation` seems sufficient.